scheduler/pkg/agent/rproxy_grpc.go

```diff
@@ -119,6 +119,7 @@ func (rp *reverseGRPCProxy) Start() error {
 		opts = append(opts, grpc.Creds(rp.tlsOptions.Cert.CreateServerTransportCredentials()))
 	}
 	opts = append(opts, grpc.MaxConcurrentStreams(grpcProxyMaxConcurrentStreams))
+	opts = append(opts, grpc.MaxRecvMsgSize(util.GrpcMaxMsgSizeBytes))
```

Comment:
why we are not setting `MaxCallSendMsgSize` as the other ones here as well?

Reply:
will update